### ISSUE TITLE
Monster: Skeletal Juggernaut(Auto Effects)

### ISSUE THREAD

**Issue by MotoMoto1234** [MrPrimate/ddb-importer#360]
If these changes are hard to follow I have them shared here:  https://discord.com/channels/915186263609454632/1121490044402802698/1121490044402802698

```
Falling Apart:
  Effects Tab:
    Falling Apart Effect:
      Details Tab:
        Effect Icon: icons/skills/wounds/bone-broken-knee-beam.webp
        Transfer to actor on item equip: True
        Stackable: Effects do not stack by NAME
      Effects Tab:
        Attribute Key: flags.midi-qol.OverTime
        Change Mode: Override
        Effect Value: turn=start, damageRoll=10, damageType=none, condition=@attributes.hp.value > 0 && @attributes.hp.value < @attributes.hp.max, label=Falling Apart
```
```
Avalanche of Bones:
  Item Details Tab:
    Target: 10 Feet Creature
    Range: Self
    Duration: Special
    Midi Check Boxes: Half Dam Save
  Effects Tab:
    Avalanche of Bones Effect:
      Details Tab:
        Effect Icon: icons/commodities/bones/bones-stack-tan.webp
      Duration Tab:
        Effect Duration(Seconds): 9999
        Effect Duration(turns): 99
          (This effect needs to be visible so the player knows they need to stand up)
      Effects Tab:
        Attribute Key: macro.CE
        Change Mode: Custom
        Effect Value: Prone
```
```
Disassemble:
  Effects Tab:
    Disassemble Effect:
      Details Tab:
        Effect Icon: icons/magic/death/undead-skeleton-deformed-red.webp
        Transfer to actor on item equip: True
      Duration Tab:
        Special Duration: Zero HP: expires if the character reaches 0 HP
      Effects Tab:
        Attribute Key: macro.itemMacro
        Change Mode: Custom
        Effect Value: Empty
```
Item Macro for Disassemble:
```js
const active = (module) => game.modules.get(module)?.active;

if ( !(active('sequencer') && active('warpgate')) ) return;
if ( !(active('jb2a_patreon') || active('JB2A_DnD5e')) ) return;

if ( args[ 0 ] === "off" ) {
  const imgs = await game.actors.getName("Skeleton").getTokenImages()
  await Promise.all(imgs.map(i => loadTexture(i)));
  for(let i = 0; i < 12; i++){
    const [spawn] = await warpgate.spawn("Skeleton");
    new Sequence().effect().file("jb2a.misty_step.01.blue").atLocation(spawn).play()
  }
}
```

**Comment by MrPrimate** [comment]
Thanks
